src/components/common/AddEditBusCorp.vue

```diff
@@ -0,0 +1,663 @@
+<template>
```

Comment:
New file. I copied this from AddEditOrgPerson. I still have a ton of work to do in here.